openapi.yaml

```diff
@@ -7324,6 +7331,21 @@ paths:
                     A label used to group Linodes for display. Linodes are not
                     required to have a group.
                   example: Linode-Group
+                placement_group:
+                  type: object
+                  description: |
+                    Include this to assign this Linode to an existing [placement group](/docs/products/compute/compute-instances/guides/placement-groups/). Consider these points when cloning:
+                    
+                    * If the Linode you're cloning exists in a placement group, the API won't automatically add the cloned instance to the same placement group. You need to specify a placement group to add the clone to.
+                    * The target placement group needs to be in the same `region` set for this Linode.
+                    * The placement group needs to have capacity. Run the [Region View](/docs/api/regions/#region-view) operation and store the `maximum_linodes_per_pg` value to know the Linode limit per placement group. You can then run the [Placement Group View](/docs/api/placement-groups/#placement-group-view) operation to review the Linodes in that group.
+                  required:
+                  - id
```

Comment:
This is technically correct, although the CLI gets a little confused when flattening the argument and thinks this field is always required when creating an instance. It might make sense to drop it for the time being, otherwise I can maybe look into applying a fix in the CLI 👍

Reply:
@lgarber-akamai ... Are you saying that I should just drop `required: - id`?

Reply:
Yup!